src/ucp/core/ucp_context.c

```diff
@@ -1302,6 +1302,14 @@ static ucs_status_t ucp_fill_config(ucp_context_h context,
     ucs_debug("Estimated number of endpoints per node is %d",
               context->config.est_num_ppn);
 
+    if (context->config.ext.bcopy_bw == UCS_BANDWIDTH_AUTO) {
+        /* bcopy_bw wasn't set via the env variable. Calculate the value */
+        context->config.ext.bcopy_bw =
+            ucs_cpu_est_bcopy_bw[ucs_arch_get_cpu_vendor()];
```

Comment:
i would add ucs API: ucs_get_memcpy_bw() - instead of array access

Reply:
done:
added `ucs_cpu_get_memcpy_bw()` and used here